__tests__/components/Comments/WriteComment.test.js

```diff
@@ -53,7 +53,7 @@ describe('WriteComment should run correctly', () => {
       flex: 1,
       backgroundColor: COLORS.white,
       width: '100%',
-      paddingRight: 10,
+      paddingRight: 20,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The padding on the right has been increased from 10 to 20. This change might affect the layout of the component, especially on smaller screens. Please ensure that this change doesn't cause any UI issues on different screen sizes.
```diff
- paddingRight: 10,
+ paddingRight: 20,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->